packages/compass-sidebar/src/components/legacy/sidebar-databases-navigation.tsx

```diff
@@ -47,13 +47,13 @@ function SidebarDatabasesNavigation({
       // TODO: COMPASS-7718 to use connectionId for new tabs
```

Comment:
```suggestion
```

Reply:
I thought I removed it - maybe I did it in the new sidebar.